### ISSUE TITLE
Check links before starting the site

### ISSUE THREAD

**Issue by JoelMarcey** [facebook/Docusaurus#210]
Can we have it where if links are known to be bad, that when you run `yarn start` to run a site, it fails with a bad links message?

**Comment by JoelMarcey** [comment]
Maybe `textlint` and friends described in https://github.com/facebook/Docusaurus/issues/210 could fix this?

**Comment by azu** [comment]
Probably, correct link is https://github.com/facebook/Docusaurus/pull/302

**Comment by JoelMarcey** [comment]
Oh right. Nice catch. I updated my comment :)

**Comment by endiliey** [comment]
I think linting should be done via user side. It's not nice to crawl & check for external link.

**Comment by JoelMarcey** [comment]
Not saying we shouldn't close this necessarily -- but I think this was in reference to *internal* links within the docs. To make sure we didn't type anything bad in [xxx](link-to-local-doc.md) or something.

**Comment by endiliey** [comment]
I think i missed that. We could file a new issue if needed 😊. Old issue like this usually isnt seen by contributor, so new issue can kinda bump things up

**Comment by JoelMarcey** [comment]
Sure. Let's file a new issue.
